src/main/java/com/cloudbees/hudson/plugins/folder/computed/DeletingChildrenException.java

```diff
@@ -0,0 +1,20 @@
+package com.cloudbees.hudson.plugins.folder.computed;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DeletingChildrenException extends RuntimeException {
+
+    private List<IOException> exceptions;
+
+    public DeletingChildrenException(List<IOException> exceptions) {
+        this.exceptions = exceptions;
+    }
+
+    @Override
+    public String getMessage() {
+        return "Impossible to delete some children when computing the folder: \n" +
+                String.join("\n", exceptions.stream().map(e -> e.getMessage()).collect(Collectors.toList()));
```

Comment:
Do you think adding some indentation in the message could visually help to understand the end of the list?
```suggestion
                String.join("\n\t", exceptions.stream().map(e -> e.getMessage()).collect(Collectors.toList()));
```

Reply:
If you have hundreds of items failing to be deleted this will just clog up the logs making it hard to read and possibly filling up the buffer. You should at least make the list of messages distinct (though they might not be since they might contain the path of the file)

Reply:
@rsandell so do you think the message would be better removed?

Reply:
Yes, If you add the first in the list as the cause then we can still see that in the stacktrace.
Or if you'd like take for example only the first three distinct messages.